### PR TITLE
Enable tests rt_sigaction02, rt_sigqueuinfo01, sigwaitinfo01

### PR DESCRIPTION
**Pull request by KenGordon**
Enable tests rt_sigaction02, rt_sigqueuinfo01, sigwaitinfo01 following signaling improvements (see issue 709).

### CODE REVIEW

**Review by SeanTAllen — APPROVED** [review]

File: tests/ltp/ltp-batch2/ltp_disabled_tests.txt [hunk 1]
```diff
@@ -898,7 +898,7 @@
 #/ltp/testcases/kernel/syscalls/sigprocmask/sigprocmask01
 /ltp/testcases/kernel/syscalls/sigrelse/sigrelse01
 /ltp/testcases/kernel/syscalls/sigsuspend/sigsuspend01
-/ltp/testcases/kernel/syscalls/sigwaitinfo/sigwaitinfo01
```
Comment (vtikoo) [inline tests/ltp/ltp-batch2/ltp_disabled_tests.txt]: Is this intentional?

**Review by vtikoo — APPROVED** [review]

File: tests/ltp/ltp-batch2/ltp_disabled_tests.txt (hunk 1 above)
Comment (KenGordon) [reply tests/ltp/ltp-batch2/ltp_disabled_tests.txt]: No. I will fix it.

**Comment by davidchisnall** [comment]
@KenGordon, this has broken CI, please can you revert it?

**Comment by KenGordon** [comment]
> @KenGordon, this has broken CI, please can you revert it?

Fixed in PR https://github.com/lsds/sgx-lkl/pull/837 just for this.

**Pull request merged**